app/views/products/_modal_window.html.haml

```diff
@@ -0,0 +1,3 @@
+#modal-window.modal.hide.fade{"aria-hidden" => "true", "aria-labelledby" => "myModalLabel", role: "dialog"}
+  .modal-dialog{role: "document"}
+    .modal-content 
```

Comment:
why do you render this empty window instead of rendering 'modal' partial? Or I didn't noticed something?